docs/configuration/configuration.md

```diff
@@ -58,6 +58,7 @@ An example [oauth2-proxy.cfg]({{ site.gitweb }}/contrib/oauth2-proxy.cfg.example
 | `--github-token` | string | the token to use when verifying repository collaborators (must have push access to the repository) | |
 | `--github-user` | string \| list | To allow users to login by username even if they do not belong to the specified org and team or collaborators | |
 | `--gitlab-group` | string | restrict logins to members of any of these groups (slug), separated by a comma | |
+| `--gitlab-project` | string | restrict logins to members of any of these project | |
```

Comment:
This just merged and fixes an issue with the `--gitlab-group` config setup (singular vs plural and `StringSlice` flag): https://github.com/oauth2-proxy/oauth2-proxy/pull/639

I think you likely want to mirror those changes for projects assuming you also want to allow multiples.

Reply:
I made the changes for multiple groups, not sure about how you want to log failed attempts. Can you give me an advice for this ? Thanks a lot.